test/message.spec.js

```diff
@@ -0,0 +1,126 @@
+var Promise = require("bluebird");
+var expect = require("chai").expect;
+var supertest = require("supertest-promised");
+var parseXML = require("xml-parser");
+var find = require("lodash/find");
+var app = require("../app");
+var SurveyResponse = require("../models/SurveyResponse");
+var agent = supertest(app);
+
+describe("POST /message", function() {
+  beforeEach(function() {
+    return SurveyResponse.remove({})
+    .then(() => console.log("### delete all"))
+  });
+
+  it("Should return a greeting/question on first survey attempt.", function() {
```

Comment:
Nitpick: As a personal preference for writing examples I would write this like:

```javascript
it("returns a greeting and a question on first survey attempt", function() {
//  ^ notice that I don't use `should`
```

Additionally there is no need to capitalize the first word, think in the context that is provided by `it`. If you want to discuss about this, please ping me.

I'd love to hear your thoughts on this.